homeassistant/components/motioneye/config_flow.py

```diff
@@ -123,8 +133,12 @@ def _get_form(
         # at least prevent entries with the same motionEye URL.
         self._async_abort_entries_match({CONF_URL: user_input[CONF_URL]})
 
+        title = user_input[CONF_URL]
+        if self._hassio_discovery:
+            title = self._hassio_discovery["addon"]
```

Comment:
Since the real value of 'addon' appears to be "motionEye" (as per your screenshot), I think this title probably needs to change. Imagine a case where there's a separate motionEye server and the addon, the titles shown on the frontend would be:

 -  motionEye
 -  http://seperate-server/

.. which is fairly confusing. I think `f"{self._hassio_discovery["addon"]} add-on}"` or equivalent is probably clearer.

Reply:
Lets remove the name, and just "Add-on". The discovery service is limited to 1 add-on per service type anyways, thus it cannot collide.